components/src/PaymentForm/paymentform.main.tsx

```diff
@@ -207,57 +208,66 @@ class PaymentFormMain extends React.Component<PaymentFormMainProps, PaymentFormM
                   .filter(k => lambdaResponse.hasOwnProperty(k))
                   .map(k => `${encodeURIComponent(k)}=${encodeURIComponent(lambdaResponse[k])}`)
                   .join('&');
-                const cardData = `&bill_to_forename=${name[0]}&bill_to_surname=${name[1]}&card_type=${cardType}&card_number=${cardNumber}&card_expiry_date=${formatedExpiryMonth}-${expiryYear}&card_cvn=${securityCode}`;
+                const cardData = `&receipt_page_redirect_url=http://localhost:8080/&cancel_page_redirect_url=http://localhost:8080/&bill_to_forename=${name[0]}&bill_to_surname=${name[1]}&card_type=${cardType}&card_number=${cardNumber}&card_expiry_date=${formatedExpiryMonth}-${expiryYear}&card_cvn=${securityCode}`;
```

Comment:
These should be set from the lambda, and point to the lambda to perform the redirect